examples/multi_window_ref_app/lib/app/popup_window_content.dart

```diff
@@ -0,0 +1,79 @@
+import 'package:flutter/material.dart';
+import 'package:multi_window_ref_app/app/child_window_renderer.dart';
+import 'package:multi_window_ref_app/app/positioner_settings.dart';
+import 'package:multi_window_ref_app/app/window_controller_text.dart';
+import 'window_settings.dart';
+import 'window_manager_model.dart';
+
+class PopupWindowContent extends StatelessWidget {
+  const PopupWindowContent(
+      {super.key,
+      required this.controller,
+      required this.windowSettings,
+      required this.positionerSettingsModifier,
+      required this.windowManagerModel});
+
+  final PopupWindowController controller;
+  final WindowSettings windowSettings;
+  final PositionerSettingsModifier positionerSettingsModifier;
+  final WindowManagerModel windowManagerModel;
+
+  @override
+  Widget build(BuildContext context) {
+    return ViewAnchor(
+        view: ChildWindowRenderer(
+            windowManagerModel: windowManagerModel,
+            windowSettings: windowSettings,
+            positionerSettingsModifier: positionerSettingsModifier,
+            controller: controller),
+        child: Scaffold(
+            backgroundColor: Colors.transparent,
+            body: SingleChildScrollView(
+                child: Center(
+                    child: Container(
+              decoration: BoxDecoration(
+                gradient: LinearGradient(
+                  begin: Alignment.topCenter,
+                  end: Alignment.bottomCenter,
+                  colors: [
+                    Theme.of(context).colorScheme.primary,
+                    Theme.of(context).colorScheme.secondary,
+                  ],
+                  stops: const [0.0, 1.0],
+                ),
+                borderRadius: BorderRadius.circular(12.0),
+              ),
+              child: ClipRRect(
+                borderRadius: BorderRadius.circular(12.0),
+                child: Padding(
+                  padding: const EdgeInsets.all(16.0),
+                  child: Column(
+                    mainAxisSize: MainAxisSize.min,
```

Comment:
```suggestion
                    mainAxisSize: MainAxisSize.max,
```
This ensures that the `Column` expands so the rounded rectangle with the gradient background fills the view.